### PR TITLE
Add Night Craft quest from PotC 1.03

### PR DESCRIPTION
**Pull request by Hammie**

### CODE REVIEW

File: PROGRAM/Storyline/standard/quests/quests_reaction.c [hunk 1]
```diff
@@ -6490,13 +6501,7 @@ void QuestComplete(string sQuestName)
...
 		break;
 
-		case "turpin_cabanel_fight":
-			LAi_group_MoveCharacter(characterFromID("Turpin Cabanel"), "FRANCE_SOLDIERS");
-			LAi_group_SetRelation("FRANCE_SOLDIERS", LAI_GROUP_PLAYER, LAI_GROUP_ENEMY);
-			if(GetRMRelation(PChar, FRANCE) > REL_WAR) SetRMRelation(PChar, FRANCE, REL_WAR); // RM
-			LeaveService(PChar, FRANCE, true); // RM
-			ChangeCharacterReputation(pchar, -2);
-		break;
+// GR: -->	case "turpin_cabanel_fight": moved to "quests_side.c"
```
Comment (Hammie) [inline PROGRAM/Storyline/standard/quests/quests_reaction.c:6504]: I think the modified `quests_side.c` is missing.

Should I just revert this change?

**Review by Hammie — COMMENTED** [review]
@AkrimalS can you check this?

File: PROGRAM/Storyline/standard/quests/quests_reaction.c (hunk 1 above)
Comment (AkrimalS) [reply PROGRAM/Storyline/standard/quests/quests_reaction.c:6504]: I think we need to wait until Monday and Gray Roger will provide it. I wrote to him about this.
